src/utilities/astFromValue.js

```diff
@@ -24,7 +24,11 @@ import {
 } from '../type/definition';
 
 /**
- * Produces a GraphQL Value AST given a JavaScript value.
+ * Produces a GraphQL Value AST given a JavaScript object.
+ * Function will match JavaScript/JSON values to GraphQL AST schema format
+ * by using suggested GraphQLInputType. For example:
+ *
+ *  `astFromValue("value", GraphQLString)`
```

Comment:
We use different format for code samples, e.g.:
https://github.com/graphql/graphql-js/blob/2e1becb22e7ac37a9ed5b665136b92f9d1665c5e/src/jsutils/keyMap.js#L9-L27

Can you please update this line to match?